src/briefcase/__main__.py

```diff
@@ -15,8 +15,9 @@
 def main():
     result = 0
     command = None
-    logger = Log()
-    console = Console()
+    printer = Printer()
+    console = Console(printer=printer)
+    logger = Log(printer=printer)
```

Comment:
nitpick: `Printer` doesn't really need to be exposed up here in main; a console initialization function could wrap all this (and future logic) and return just the `console` and `logger`. (although, mostly indifferent if this happens now...or when future logic is added)

Reply:
Possibly - but that likely requires a either more comprehensive refactor so that Log and Console are more closely bound (or even merged entirely); introducing another wrapper object, or a single purpose "construct both" method, that really only avoids creating the printer. At least for now, my inclination is to leave this as is, and be explicit about the fact that there's a single printer, which both the log and the console may write to.